README.md

```diff
@@ -1,17 +1,17 @@
 # Citrea
 
-**The first rollup that enhances the capabilities of Bitcoin blockspace with zero-knowledge technology, now [live on Devnet](https://www.blog.citrea.xyz/citrea-public-devnet-live/)! 🎉🍊🍋**
+**The first rollup that enhances the capabilities of Bitcoin blockspace with zero-knowledge technology, now [live on Bitcoin Testnet](https://www.blog.citrea.xyz/citrea-testnet-live-on-bitcoin-testnet4)! 🎉🍊🍋**
 
 ![](resources/assets/banner.png)
 
 > [!WARNING]
-> Citrea is still work-in-progress as a public devnet. While it's launched as a public devnet (on top of a custom Bitcoin Signet), it has not been audited and many features are under heavy development. Citrea's proper integration with [Bitcoin](https://github.com/bitcoin/bitcoin) and [Clementine](https://github.com/chainwayxyz/clementine) is still ongoing. \
+> Citrea is still work-in-progress as a testnet. While the devnet phase is successfully completed and it's now live on Bitcoin Testnet4, it has not been audited and many features are under heavy development. Citrea's proper integration with [Bitcoin Mainnet](https://github.com/bitcoin/bitcoin) and [Clementine](https://github.com/chainwayxyz/clementine) are still ongoing. \
```

Comment:
~While the devnet phase is successfully completed and it's now live on Bitcoin Testnet4, it has not been audited and many features are under heavy development.~

Citrea testnet is launched on top of Bitcoin Testnet4. Please note that Citrea has not been audited and many features are under heavy development.